service/routes.py

```diff
@@ -43,8 +43,25 @@ def create_order():
     order = Order()
     order.deserialize(request.get_json())
     order.create()
+    location_url = url_for("read_orders", order_id=order.id, _external=True)
     app.logger.info("Order with ID [%s] created.", order.id)
-    return jsonify(order.serialize()), status.HTTP_201_CREATED
+    return jsonify(order.serialize()), status.HTTP_201_CREATED, {"Location": location_url}
+
+######################################################################
+# READ AN ORDER
+######################################################################
+@app.route("/orders/<int:order_id>", methods=["GET"])
+def read_orders(order_id):
```

Comment:
instead of using the name read_orders, can we make it get_order as we are returning a single order in this API?

Reply:
good suggestion, done